### PR TITLE
FLYWOOF411HD

### PR DESCRIPTION
**Pull request by tstibor**
* @ Steve_EOD flew `FLYWOOF411HD` without problems.
* may need deep testing.

### CODE REVIEW

File: src/main/target/FLYWOOF411EVO_HD/target.c [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    DEF_TIM(TIM1, CH1, PA8,  TIM_USE_MOTOR,  0, 1), // S1_OUT 2,1
+    DEF_TIM(TIM1, CH2, PA9,  TIM_USE_MOTOR,  0, 1), // S2_OUT 2,2
+    DEF_TIM(TIM1, CH3, PA10, TIM_USE_MOTOR,  0, 0), // S3_OUT 2,6
+    DEF_TIM(TIM2, CH3, PB10, TIM_USE_MOTOR,  0, 0), // S4 - D(1,1)
+
+    DEF_TIM(TIM3, CH4, PB1,  TIM_USE_ANY,    0, 0), //  RSSI   1,2
+    DEF_TIM(TIM3, CH1, PB4,  TIM_USE_ANY,    0, 0), //  SOFTSERIAL_1_TX_PIN 1-4
+    DEF_TIM(TIM3, CH3, PB0,  TIM_USE_ANY,    0, 0), //  SOFTSERIAL_1_RX_PIN 1-7
+    DEF_TIM(TIM5, CH4, PA3,  TIM_USE_ANY,    0, 1), //  RX2 1,0
```
Comment (BeauBrewski) [inline src/main/target/FLYWOOF411EVO_HD/target.c:41]: Line 41 needs removed, this definition is not part of the config file for this target

File: src/main/target/FLYWOOF411EVO_HD/target.h [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+#define CURRENT_METER_SCALE_DEFAULT         170
+#define DEFAULT_VOLTAGE_METER_SOURCE    VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SOURCE    CURRENT_METER_ADC
+
+#define TARGET_IO_PORTA         0xffff
+#define TARGET_IO_PORTB         0xffff
+#define TARGET_IO_PORTC         0xffff
+#define TARGET_IO_PORTD         (BIT(2))
+
+#define USABLE_TIMER_CHANNEL_COUNT 10
```
Comment (BeauBrewski) [inline src/main/target/FLYWOOF411EVO_HD/target.h:148]: According to to config for this target, there are 9 timer channels

File: src/main/target/FLYWOOF411EVO_HD/target.h [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+#define USED_TIMERS             ( TIM_N(1)|TIM_N(2)|TIM_N(3)|TIM_N(5)|TIM_N(9) )
```
Comment (BeauBrewski) [inline src/main/target/FLYWOOF411EVO_HD/target.h:149]: TIM_N(5) is not used for this target according to the config.

File: src/main/target/FLYWOOF411EVO_HD/target.h [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+#define SPI1_MOSI_PIN          PA7
+
+#define USE_SPI_DEVICE_2
+#define SPI2_SCK_PIN               PB13
+#define SPI2_MISO_PIN          PB14
+#define SPI2_MOSI_PIN          PB15
+
+// *************** Gyro & ACC **********************
+#define MPU6000_CS_PIN          PA4
+#define ICM20689_CS_PIN         PA4
```
Comment (BeauBrewski) [inline src/main/target/FLYWOOF411EVO_HD/target.h:48]: I don't think ICM20689 definitions are required for this target. Pretty sure its MPU6000 only, but don't want them removed without testing and the only tester says it flies great now.

File: src/main/target/FLYWOOF411EVO_HD/target.h [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+#define USE_UART1
+#define UART1_RX_PIN            PB7
+#define UART1_TX_PIN            PB6
+
+#define USE_UART2
+#define UART2_RX_PIN            PA3
+#define UART2_TX_PIN            PA2
+
+#define USE_SOFTSERIAL1
+#define USE_SOFTSERIAL2
```
Comment (BeauBrewski) [inline src/main/target/FLYWOOF411EVO_HD/target.h:100]: Manufacturer description claims only 1 softserial

**Review by BeauBrewski — COMMENTED** [review]
I've nitpicked on a few lines of unnecessary code that don't appear to affect current functionality of this target according to the 1 tester. Not sure if they will cause possible issues, or can just be ignored and left unchanged. 🤷

**Comment by nerdCopter** [comment]
@BeauBrewski , if you like to take this over and retain @tstibor 's contributions, you may:
```
gh pr checkout 718
git checkout -b newBranchName
```
then make mods; perform testing; push branch; and make new PR as usual.  thank you!

**Pull request closed without merging**